example_dags/anyscale_job.py

```diff
@@ -44,6 +46,77 @@
     dag=dag,
 )
 
+JOB_QUEUE_NAME = "test-job-queue-180s-idle-timeout"
+
+submit_anyscale_job_with_new_job_queue = SubmitAnyscaleJob(
+    task_id="submit_anyscale_job_with_new_job_queue",
+    conn_id=ANYSCALE_CONN_ID,
+    name="AstroJobWithJobQueue",
+    image_uri="anyscale/image/airflow-integration-testing:1",
+    compute_config="airflow-integration-testing:1",
+    working_dir=str(FOLDER_PATH),
+    entrypoint="python ray-job.py",
+    max_retries=1,
+    job_timeout_seconds=3000,
+    poll_interval=30,
+    dag=dag,
+    job_queue_config=JobQueueConfig(
+        priority=100,
+        job_queue_spec=JobQueueSpec(
+            name=JOB_QUEUE_NAME,
+            execution_mode=JobQueueExecutionMode.PRIORITY,
+            max_concurrency=5,
+            idle_timeout_s=180,
+        ),
+    ),
+)
+
+submit_anyscale_job_with_existing_job_queue = SubmitAnyscaleJob(
+    task_id="submit_anyscale_job_with_existing_job_queue",
+    conn_id=ANYSCALE_CONN_ID,
+    name="AstroJobWithJobQueue",
+    image_uri="anyscale/image/airflow-integration-testing:1",
+    compute_config="airflow-integration-testing:1",
+    working_dir=str(FOLDER_PATH),
+    entrypoint="python ray-job.py",
+    max_retries=1,
+    job_timeout_seconds=3000,
+    poll_interval=30,
+    dag=dag,
+    job_queue_config=JobQueueConfig(
+        priority=100,
+        # NOTE: This will reuse the existing job queue given it has the same spec
+        job_queue_spec=JobQueueSpec(
+            name=JOB_QUEUE_NAME,
+            execution_mode=JobQueueExecutionMode.PRIORITY,
+            max_concurrency=5,
+            idle_timeout_s=180,
+        ),
+    ),
+)
+
+submit_another_anyscale_job_with_existing_job_queue = SubmitAnyscaleJob(
+    task_id="submit_another_anyscale_job_with_existing_job_queue",
+    conn_id=ANYSCALE_CONN_ID,
+    name="AstroJobWithJobQueue",
+    image_uri="anyscale/image/airflow-integration-testing:1",
+    compute_config="airflow-integration-testing:1",
+    working_dir=str(FOLDER_PATH),
+    entrypoint="python ray-job.py",
+    max_retries=1,
+    job_timeout_seconds=3000,
+    poll_interval=30,
+    dag=dag,
+    job_queue_config=JobQueueConfig(
+        priority=100,
+        target_job_queue_name=JOB_QUEUE_NAME,
+    ),
+)
+
 
 # Defining the task sequence
 submit_anyscale_job
+submit_anyscale_job_with_new_job_queue >> [
+    submit_anyscale_job_with_existing_job_queue,
+    submit_another_anyscale_job_with_existing_job_queue,
+]
```

Comment:
```suggestion
]

```
Maybe we should add a pre-commit hook in future PR to add newline at the end